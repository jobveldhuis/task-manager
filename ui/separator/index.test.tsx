import { Separator } from "./index";
import { render } from "@testing-library/react-native";

describe("Separator", () => {
  const component = <Separator />;

  it("should match snapshot", async () => {
    expect(render(component).toJSON()).toMatchSnapshot();
  });
});
