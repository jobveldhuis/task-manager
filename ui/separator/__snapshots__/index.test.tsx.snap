// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should match snapshot 1`] = `
<View
  style={
    [
      {
        "alignSelf": "stretch",
        "borderBottomColor": "#000",
        "borderBottomWidth": 1,
      },
      undefined,
    ]
  }
/>
`;

exports[`Text should match snapshot 1`] = `
<View
  style={
    [
      {
        "alignSelf": "stretch",
        "borderBottomColor": "#000",
        "borderBottomWidth": 1,
      },
      undefined,
    ]
  }
/>
`;
